src/main/java/seedu/duke/template/Template.java

```diff
@@ -0,0 +1,11 @@
+package seedu.duke.template;
+
+import seedu.duke.component.VoltageSource;
+
+public class Template {
+    protected VoltageSource initialPowerSupply;
```

Comment:
Same, a getter for this could be used later?

Reply:
Noted! Will add in all getters.